{
  "version": 3,
  "sources": [
    "../contract.py"
  ],
  "mappings": ";;;;;;;;;;;;;;;;;;;AAQA;;AAAA;AAAA;AAAA;;AAAA;;;AAAA;;;;;;AAAA;;;AAAA;;;;AAAA;AAKK;;;AAAA;AAAA;;AAAA;AAAA;AAAA;AAAA;AAAA;;AAAA;AAAA;AAAA;;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;;;AAAA;AAAA;AAAA;AAAA;AAAA;AAEU;AAAA;AAAA;AAAA;AAAA;;AAAA;AAMA;;;;;;AAAf;AAAA;;AAAA;AAAA;;;AACuC;;AAAA;;;AAAA;;AAAA;AAAA;AAAA;AAAA;AAAA;AAMT;;AAAA;AAAT;;;AAAA;;AAAA;AAAA;AAAV;AAAA;;AAPI;AAAA;AAAA;;;;;;AAOkE;;AAAA;;;;AAblE;;AAAA;;AAAA;AAFV;;;AAAA;AAAA;;AAAA;AAAA;;;;;;AAAA;AAAA;AAAA;AAAA;AAAA;AALL;;;;;;AAAA;;;AAAA;;;;AAAA;AACK;;;AAAA;AAAA;AAAA;AAAA;AAAA;AAEG;AAAA;AAAA;AAFH;AAAA",
  "op_pc_offset": 0,
  "pc_events": {
    "22": {
      "error": "OnCompletion must be NoOp"
    },
    "50": {
      "error": "invalid array length header"
    },
    "62": {
      "error": "invalid number of bytes for arc4.dynamic_array<arc4.static_array<arc4.uint8, 32>>"
    },
    "70": {
      "error": "invalid number of bytes for arc4.static_array<arc4.uint8, 32>"
    },
    "77": {
      "error": "check self.root exists"
    },
    "101": {
      "error": "index access is out of bounds"
    },
    "174": {
      "error": "invalid number of bytes for arc4.static_array<arc4.uint8, 32>"
    }
  }
}